.gitattributes

```diff
@@ -23,19 +23,20 @@
 /.eslintrc                       export-ignore
 /.gitattributes                  export-ignore
 /.gitignore                      export-ignore
+/.nvmrc                          export-ignore
 /.phpcs.xml.dist                 export-ignore
 /.prettierrc                     export-ignore
-/.nvmrc                          export-ignore
```

Comment:
Could we revert the changes that are not strictly necessary? `.nvmrc`, `package.json` and others do not need to be shuffled around.

Reply:
In my opinion, in small PRs like these such simple changes are not an issue. Having said that, I've reverted the alphabetic reordering.